File: C#/journeyBegins.cs
int add(int param1, int param2) {
    return param1 + param2;
}

int centuryFromYear(int year) {
    if (year%100 == 0) {
        return year/100;
    } else {
        return (int) (Math.Floor(year/100.0 + 1));
    }
}

bool checkPalindrome(string inputString) {
 string reverseString = new string(inputString.ToCharArray().Reverse().ToArray());
 return inputString == reverseString;
}

File: JS/journeyBegins.js
add = (param1, param2) => {
    return param1 + param2
}

centuryFromYear = (year) => {
    if (year % 100 === 0) {
        return Math.floor(year/100)
    } else {
        return Math.floor((year/100) + 1)
    }
}

checkPalindrome = (inputString) => {
    let reverseString = ""
    for ( let i = inputString.length - 1; i >= 0; i--) {
        reverseString += inputString[i]
    }
    console.log(reverseString)
    if (inputString === reverseString) {
        return true
    } else {
        return false
    }
}
